abstract-chained-batch.js

```diff
@@ -43,21 +43,25 @@ AbstractChainedBatch.prototype.put = function (key, value, options) {
   const err = this.db._checkKey(key) || this.db._checkValue(value)
   if (err) throw err
 
-  const keyEncoding = this.db.keyEncoding(options && options.keyEncoding)
-  const valueEncoding = this.db.valueEncoding(options && options.valueEncoding)
+  // TODO: docs, types and tests
```

Comment:
Is this a draft PR?

Reply:
100% of the preexisting API is covered by docs and tests. But the new `sublevel` option on batch is not covered yet. Not a blocker, I figured.